src/common/auto-controlled-input.js

```diff
@@ -8,14 +8,14 @@ const __DEV__ = process.env.NODE_ENV !== 'production'
 
 // This decorator can be used on a controlled input component to make
 // it able to automatically handled the uncontrolled mode.
-export default () => ControlledInput => {
+export default getEmptyValue => ControlledInput => {
```

Comment:
The sole param should be an options object or an options factory which will be called with the initial props to create the options.